### PR TITLE
UGENE-6996. Add Russian translation for some context menu for MSA editor

### PR DESCRIPTION
**Pull request by yalgaer**

### CODE REVIEW

**Review by DmitriiSukhomlinov — DISMISSED** [review]
Approved, but what about the other unfinished translations?

File: src/corelibs/U2View/transl/russian.ts [hunk 1]
```diff
@@ -2259,7 +2259,7 @@ Double-click to collapse the branch</source>
     <message>
         <location filename="../src/ov_sequence/ADVClipboard.cpp" line="280"/>
         <source>Copy annotation</source>
-        <translation type="unfinished"></translation>
+        <translation>Копировать анноттацию</translation>
```
Comment (evelinabis) [inline src/corelibs/U2View/transl/russian.ts:2262]: Typo

File: src/corelibs/U2View/transl/russian.ts [hunk 2]
```diff
@@ -4570,19 +4570,19 @@ Please, check external tools in the settings.</source>
...
         <source>Export highlighting</source>
-        <translation type="unfinished">Экспорт подсвеченного</translation>
+        <translation>Экспорт подсвеченного</translation>
     </message>
     <message>
-        <location filename="../src/ov_msa/MSAEditorSequenceArea.cpp" line="877"/>
+        <location filename="../src/ov_msa/MSAEditorSequenceArea.cpp" line="901"/>
         <source>Export highlighting finished successfully</source>
-        <translation type="unfinished">Export highligtning finished successfully</translation>
+        <translation>Export highligtning finished successfully</translation>
```
Comment (evelinabis) [inline src/corelibs/U2View/transl/russian.ts:4580]: Why is the "unfinished" removed here?

**Review by evelinabis — DISMISSED** [review]

**Comment by yalgaer** [comment]
> Approved, but what about the other unfinished translations?

There are 2 kind of unfinished translations:
1. Legacy: from the old releases. These will be fixed later (I do not have any concrete date)
2. New texts: introduced in this release. These should be fixed, so we have no untranslated texts. 

I reviewed the patch again and fixed all 2.

**Pull request merged**